### PR TITLE
KS-SK1141-AS-multisites-for-WebHosting

### PR DESCRIPTION
**Pull request by Kazarus4**
Paragraph updated after techlead asking. need peer rv before translation (on the same branch)

### CODE REVIEW

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by Kazarus4** [comment]
All translations are done. This except duplication for EN subs (only ENGB is translated).

Need proof for all subs before EN duplication and merge.

**Comment by OVH-Team-Guides** [comment]
EN, ES, IT, PL, PT versions are ok
You can proceed to duplicate EN.
DE to be proofread by @tcpdumpfbacke

File: pages/web/hosting/multisites_configure_multisite/guide.es-es.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 ---
 title: 'Alojar varios sitios web en un mismo hosting'
 slug: configurar-un-multisitio-en-un-alojamiento-web
-excerpt: 'Cómo utilizar un plan de hosting para alojar varios sitios web'
+excerpt: 'Descruba cómo utilizar un plan de hosting para alojar varios sitios web'
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/multisites_configure_multisite/guide.es-es.md]:
```suggestion
excerpt: 'Descubra cómo utilizar un plan de hosting para alojar varios sitios web'
```

File: pages/web/hosting/multisites_configure_multisite/guide.es-es.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 
-**Última actualización: 22/08/2022**
+**Última actualización: 08/06/2023**
 
 ## Objetivo
 
 Es posible alojar varios sitios web en un mismo plan de hosting, aunque los dominios no estén registrados en OVHcloud.
 
-**Esta guía explica cómo utilizar un plan de hosting para alojar varios sitios web.**
+**Descruba cómo utilizar un plan de hosting para alojar varios sitios web.**
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/multisites_configure_multisite/guide.es-es.md]:
```suggestion
**Descubra cómo utilizar un plan de hosting para alojar varios sitios web.**
```

File: pages/web/hosting/multisites_configure_multisite/guide.es-us.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
 ---
 title: 'Alojar varios sitios web en un mismo hosting'
 slug: configurar-un-multisitio-en-un-alojamiento-web
-excerpt: 'Cómo utilizar un plan de hosting para alojar varios sitios web'
+excerpt: 'Descruba cómo utilizar un plan de hosting para alojar varios sitios web'
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/multisites_configure_multisite/guide.es-us.md]:
```suggestion
excerpt: 'Descubra cómo utilizar un plan de hosting para alojar varios sitios web'
```

File: pages/web/hosting/multisites_configure_multisite/guide.es-us.md [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 
-**Última actualización: 22/08/2022**
+**Última actualización: 08/06/2023**
 
 ## Objetivo
 
 Es posible alojar varios sitios web en un mismo plan de hosting, aunque los dominios no estén registrados en OVHcloud.
 
-**Esta guía explica cómo utilizar un plan de hosting para alojar varios sitios web.**
+**Descruba cómo utilizar un plan de hosting para alojar varios sitios web.**
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/multisites_configure_multisite/guide.es-us.md]:
```suggestion
**Descubra cómo utilizar un plan de hosting para alojar varios sitios web.**
```

File: pages/web/hosting/multisites_configure_multisite/guide.pt-pt.md [hunk 5]
```diff
@@ -1,17 +1,17 @@
 ---
 title: 'Partilhar o alojamento entre vários sites'
 slug: multisites-configurar-um-multisite-no-meu-alojamento-web
-excerpt: 'Saiba como alojar diferentes websites no seu alojamento web'
+excerpt: 'Descruba como alojar diferentes websites no seu alojamento web'
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/multisites_configure_multisite/guide.pt-pt.md]:
```suggestion
excerpt: 'Saiba como alojar diferentes websites no seu alojamento web'
```

**Pull request merged**